eessi/testsuite/eessi_mixin.py

```diff
@@ -64,6 +64,14 @@ def __init_subclass__(cls, **kwargs):
         cls.valid_systems = ['*']
         if not cls.time_limit:
             cls.time_limit = '1h'
+        if not cls.readonly_files:
```

Comment:
Wait, does this work? Isn't this always true, because a default is set by ReFrame? https://reframe-hpc.readthedocs.io/en/stable/regression_test_api.html#reframe.core.pipeline.RegressionTest.readonly_files